tests/static-api/_expected/v1/repos/archived_repo.json

```diff
@@ -14,11 +14,15 @@
   "branch_protections": [
     {
       "pattern": "master",
-      "ci_checks": [
-        "CI"
-      ],
       "dismiss_stale_review": false,
-      "required_approvals": 1,
+      "mode": {
+        "PrRequired": {
```

Comment:
nit: should this be serialized with snake case?

Reply:
Indeed it should! Fixed, thanks.